CMakeLists.txt

```
@@ -241,11 +243,13 @@ if(SDLSOUND_BUILD_SHARED)
     if(OS2)   # OS/2 does not support a DLL name longer than 8 characters.
         SET_TARGET_PROPERTIES(SDL2_sound PROPERTIES OUTPUT_NAME "SDL2snd")
     endif()
-    target_include_directories(SDL2_sound
-        PUBLIC $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
+    target_include_directories(SDL2_sound PUBLIC 
+        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
```

Allow `#include "SDL2/SDL_sound.h"` as well.
```suggestion
        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>
        $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/SDL2>
```

---

Done with f0b29fc0ea8c96da4198bf7adf75279acf43ad2c